src/bundle/Resources/public/js/scripts/admin.contenttype.edit.js

```diff
@@ -451,18 +451,22 @@
     editForm.addEventListener(
         'submit',
         (event) => {
-            const fieldDefinitionsCount = doc.querySelectorAll('.ibexa-collapse--field-definition').length;
+            const { submitter } = event;
 
-            validateForm();
+            if (!submitter.classList.contains('ibexa-content-type-edit__remove-draft')) {
```

Comment:
This should not be a problem here because in this case click to submit button is done via the button from context menu but I will add `Optional Chaining` it should fix possible problems in the future if this even would be emitted from another element

Reply:
Yes, but someone may have a custom script here or we may add something in the future. Adding optional chaining creates a more robust solution.

Reply:
what do you think about using the attribute `formnovalidate` for this `if` (we may have more buttons which should not validate form)?
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-formnovalidate